### DISCUSSION TITLE
Rke2 impact

### DISCUSSION THREAD

**Discussion by rashidalnaemi** [k3s-io/k3s#9970] [General]
Hello,

With the release of RKE2, it sounds like SUSE is positioning it as a replacement with words like "best of both worlds". Is that the case? It's a little disconcerting for a team considering using k3s in production.

Are there any sort of official or unofficial commitments for the continuity of this project?

Thanks.

**Comment by cwayne18** [comment]
RKE2 has been around for 3 years, k3s is not going anywhere

**Comment by brandond** [comment]
RKE2 is built on top of K3s. RKE2 executes the core Kubernetes components differently, and packages different addons (ingress, CNI, and so on) than K3s - but it is K3s at its core.

I will also note that we still maintain RKE, despite it being built on top of Docker and cri-dockerd. None of our distros are going anywhere.
